test/nn/test_pooling.py

```diff
@@ -1136,10 +1117,6 @@ def test_AvgPool3d_backward_after_cat_dim1_device(self, device):
 
     def test_pooling_size_empty(self, device):
         t = torch.rand([1, 2, 3, 4], device=device)
-        self.assertRaises(RuntimeError, lambda: F.adaptive_avg_pool1d(t, []))
-        self.assertRaises(RuntimeError, lambda: F.adaptive_avg_pool2d(t, []))
-        self.assertRaises(RuntimeError, lambda: F.adaptive_avg_pool3d(t, []))
-        self.assertRaises(RuntimeError, lambda: F.adaptive_max_pool1d(t, []))
         self.assertRaises(RuntimeError, lambda: F.adaptive_max_pool2d(t, []))
```

Comment:
Why are these error-cases left in this test?

Reply:
Oops, added these cases too. My bad missed it earlier.